source/clear-linux/tutorials/gce/gce.rst

```diff
@@ -0,0 +1,122 @@
+      :scale: 50 %
+      :alt: Sign in Google services
+
+#. Download the |CL| image archive for Google Compute Engine (GCE)
+   *clear-<release number>-gce.tar.gz*
+   from the |CL-PRJ| image repository. You don't need to uncompress the image
+   archive, the intact file will be uploaded to the Google Cloud Storage later.
+
+#. Click the *Navigation menu* icon on the upper left screen menu, select
+   the *Storage* item from the side bar on the left, and then click the
+   ``CREATE BUCKET`` button on the Cloud Storage page to create a *bucket*
+   for hosting the |CL| image source archive downloaded in the previous step.
+   The bucket name must be unique as buckets in the Cloud Storage share
+   a single global namespace.
+
+   .. figure:: figures/01-cloud-storage.png
+      :scale: 50 %
+      :alt: Browse Google Cloud Storage
+
+   .. figure:: figures/02-storage-bucket.png
+      :scale: 50 %
+      :alt: Create a bucket in the Cloud Storage
+
+   .. figure:: figures/03-create-bucket.png
+      :scale: 50 %
+      :alt: Set an unique bucket name
+
+#. Once the bucket is created, press the ``Upload files`` button
+   on the Bucket details page to upload the |CL| GCE image archive
+   to the named bucket:
+
+   .. figure:: figures/04-bucket-created.png
+      :scale: 50 %
+      :alt: Cloud Storage bucket is available for storing objects
+
+   .. figure:: figures/10-image-upload.png
+      :scale: 50 %
+      :alt: Uploading the image source archive file
+
+   .. figure:: figures/11-bucket-uploaded.png
+      :scale: 50 %
+      :alt: Image archive imported complete
+
+#. Click the *Navigation menu* icon on the upper left screen menu,
+   mouse hovers over the *Compute Engine* menu group and
+   select the *Images* item.
+   On the Compute Engine Images page, click the ``[+] CREATE IMAGE`` menu
+   item, change the image source type to *Cloud Storage file*,
+   and pick up the image archive object in the named bucket by clicking
+   the ``Browse`` button of the *Cloud Storage file* input box.
+   Accept the rest of default options, and click the ``Create`` button
+   at the bottom to add a Clear Linux GCE image to the image list.
+
+   .. figure:: figures/20-gce-image.png
+      :scale: 50 %
+      :alt: Create a Google Compute Engine image
+
+   .. figure:: figures/21-create-image.png
+      :scale: 50 %
+      :alt: Create the image using the imported image archive object
+
+   .. figure:: figures/22-image-list.png
+      :scale: 50 %
+      :alt: Clear Linux Compute Engine image is created
+
+#. After the |CL| image is imported, you can launch a VM instance running
+   |CL| by following the instructions documented in the
+   `Quick Start <https://cloud.google.com/compute/docs/quickstart-linux>`_
+   guide, except the step of selecting *Boot disk* --
+   Instead of using the stock OS images, click the *Custom images* tab
+   to select the imported Clear Linux OS GCE image.
+
+   .. figure:: figures/30-create-vm.png
```

Comment:
I suggest removing or blanking out the section that says "You have X free trial credits remaining".

Reply:
Gotcha! removed.